tool.go

```diff
@@ -498,7 +498,21 @@ func main() {
 	cmdServe.Run = func(cmd *cobra.Command, args []string) {
 		options.BuildTags = strings.Fields(*tags)
 		dirs := append(filepath.SplitList(build.Default.GOPATH), build.Default.GOROOT)
-		sourceFiles := http.FileServer(serveCommandFileSystem{options: options, dirs: dirs, sourceMaps: make(map[string][]byte)})
+		root := ""
```

Comment:
Optional style suggestion. Since you you're not trying to set a particular initial value for `root`, it might be more idiomatic to write this in the `var root string` form.